dotCMS/src/main/java/com/dotmarketing/cms/content/submit/util/CaptchaServiceSingleton.java

```diff
@@ -3,6 +3,7 @@
 import com.dotcms.repackage.com.octo.captcha.service.sound.DefaultManageableSoundCaptchaService;
 import com.dotcms.repackage.com.octo.captcha.service.sound.SoundCaptchaService;
 
+@Deprecated
 public class CaptchaServiceSingleton {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)